CMakeLists.txt

```diff
@@ -63,24 +66,33 @@ if(CMAKE_COMPILER_IS_GNUCC OR CMAKE_C_COMPILER_ID MATCHES "(Apple)?[Cc]lang")
     endif()
 
     set(OT_CFLAGS
-        $<$<COMPILE_LANGUAGE:C>:${OT_CFLAGS} -Wall -Wextra -Wshadow>
-        $<$<COMPILE_LANGUAGE:CXX>:${OT_CFLAGS} -Wall -Wextra -Wshadow -Wno-c++14-compat -fno-exceptions>
+        $<$<COMPILE_LANGUAGE:C>:${OT_CFLAGS}
+        -Wall
+        -Wextra
+        -Wshadow>
+        $<$<COMPILE_LANGUAGE:CXX>:${OT_CFLAGS}
+        -Wall
+        -Wextra
+        -Wshadow
+        -Wno-c++14-compat
+        -fno-exceptions>
     )
 endif()
 
 execute_process(
     COMMAND bash "-c" "third_party/nlbuild-autotools/repo/scripts/mkversion -b ${OT_DEFAULT_VERSION}"
     WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}
-    OUTPUT_VARIABLE OT_VERSION OUTPUT_STRIP_TRAILING_WHITESPACE
+    OUTPUT_VARIABLE OT_VERSION
+    OUTPUT_STRIP_TRAILING_WHITESPACE
 )
 message(STATUS "Version: ${OT_VERSION}")
 
-target_compile_definitions(ot-config INTERFACE
-    "PACKAGE_NAME=\"OPENTHREAD\""
-    "PACKAGE_VERSION=\"${OT_VERSION}\""
-)
```

Comment:
What about enforcing style of this kind of statement to like:
```cmake
target_compile_definitions(ot-config
    INTERFACE
        "PACKAGE_NAME=\"OPENTHREAD\""
        "PACKAGE_VERSION=\"${OT_VERSION}\""
)
```
?

Reply:
I don't think the tool support it. If you know how to make it, I'm glad to adopt this style.